pennylane/wires.py

```diff
@@ -94,9 +95,11 @@ def __len__(self):
     def __contains__(self, item):
         """Method checking if Wires object contains an object."""
         if isinstance(item, Wires):
-            item = item.tolist()
+            item = item.toset()
         # if all wires can be found in tuple, return True, else False
-        return all(wire in self.labels for wire in item)
+        items_set = set(item)
+        not_found_items = (items_set ^ set(self.labels)) & items_set
```

Comment:
Isn't this just the (non-symmetric) difference `items_set - set(self.labels)`?

https://www.linuxtopia.org/online_books/programming_books/python_programming/python_ch16s03.html

Reply:
also, should we use your new `self.toset()` here?

Reply:
Yes you are correct...